tests/mocha/server/HelpersTest.js

```diff
@@ -2,29 +2,26 @@ MochaWeb.testOnly(function() {
   describe('helpers', function() {
 
     describe('getCompetitionEvents', function() {
-      var compId = Competitions.insert({ competitionName: "Comp Etition", listed: false, startDate: new Date() });
+      beforeEach(function() {
+        this.compId = Competitions.insert({ competitionName: "Comp Etition", listed: false, startDate: new Date() });
```

Comment:
Hmm... I'm confused about scope here. Down below, we're not accessing `compId` via `this.compId`. How is this working?